pkg/daemon/cleanup/cleanup.go

```diff
@@ -81,13 +81,7 @@ func Deconfigure(cfg *rest.Config, ctx context.Context, preserveNode bool, confi
 	if err = removeServices(svcMgr, cmData.Services); err != nil {
 		return err
 	}
-
-	// Delete node if needed
-	if preserveNode || node == nil {
-		return nil
-	}
-	klog.Infof("deleting node %s", node.GetName())
-	return directClient.Delete(ctx, node)
```

Comment:
So is the node object no longer getting deleted? I don't see it being deleted by WMCO

Reply:
OK, I see it being deleted [here](https://github.com/openshift/windows-machine-config-operator/blob/014992656747dde271921e814ffd9ef5f03159e0/controllers/controllers.go#L195). Please call this out in your commit message.

Reply:
[Done](https://github.com/openshift/windows-machine-config-operator/pull/1332/commits/76280c74da1ed3ee132fa0b9b9b71211b14ad442) in this commit.
Removing directories, HNS networks and
deleting the node object will be done by WMCO.